de_web_plugin.cpp

```diff
@@ -2683,6 +2683,19 @@ void DeRestPluginPrivate::addLightNode(const deCONZ::Node *node)
             }
             lightNode.setNeedSaveDatabase(true);
         }
+        // dimmer switches
+        if (R_GetProductId(&lightNode).startsWith(QLatin1String("Tuya_DIMSWITCH")))
+        {
+            lightNode.addItem(DataTypeUInt8, RStateBri);
+            
+            ResourceItem *Type = lightNode.item(RAttrType);
```

Comment:
local variables should start with lower case letter, so either `type` or `item`.

Reply:
done